spec/requests/bimi_logo_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'BIMI logo' do
+  it 'is available' do
+    # If you're troubleshooting this spec, there's a good chance you're trying to remove a file that
+    # appears to be unused. This comment is here to assure you that it is in-fact used, referenced
+    # as part of the BIMI DMARC records associated with the Login.gov domain. The image should not
+    # be removed as long as it's referenced by those records.
+    get '/images/login-icon-bimi.svg'
+
+    expect(response.status).to eq(200)
```

Comment:
should we assert any of the things about the file that it matches the BIMI spec? is less than 32kb, doesn't have attributes on the root element, etc?

Reply:
I think it's a good idea, yeah. We have some similar patterns in `spec/svg_spec.rb`